lib/nimble_template/addons/variants/phoenix/mix_release.ex

```diff
@@ -23,4 +25,61 @@ defmodule NimbleTemplate.Addons.Phoenix.MixRelease do
 
     project
   end
+
+  defp edit_files(%{otp_app: otp_app, web_module: web_module} = project) do
+    Generator.delete_content(
+      "config/runtime.exs",
+      """
+      # Start the phoenix server if environment is set and running in a release
```

Comment:
do you mean like this?
```suggestion
      # Start the phoenix server if the `PHP_SERVER` environment variable is set and running in a release
```

Reply:
This is the delete content command @longnd and that content comes from the Phoenix :D